components/user/UserRepos.js

```diff
@@ -81,10 +115,47 @@ export default function UserRepos({ manage = false, confirmDelete, repos }) {
   );
 
   return (
+    <>
+    <div className="flex gap-4">
+    {!reorder && (
+          <Button
+            onClick={() => setReorder(true)}
+            disabled={reposList.length < 2}
+          >
+            <ArrowPathIcon className="h-5 w-5 mr-2" />
+            REORDER
+          </Button>
+        )}
+        {reorder && (
+          <Button
+            onClick={() => {
+              setReorder(false);
+              setReposList(reposListPrevious);
+            }}
+          >
+            CANCEL
+          </Button>
+        )}
+        {reorder && (
+            <Button primary={true} onClick={() => saveOrder()}>
+              SAVE
+            </Button>
+          )}
+    </div>
     <ul role="list" className="divide-y divide-primary-low">
-      {repos.map((repo) => (
+      <ReactSortable
+          list={reposList}
+          setList={setReposList}
+          disabled={!reorder}
+          ghostClass="border-2"
+          chosenClass="border-dashed"
+          dragClass="border-red-500"
+        >
```

Comment:
some tests failed because `<li>` elements must be contained in a `<ul>` or `<ol>`, but here `<li>` contained by `<div>` element.
to fix this please follow the `react-sortablejs` docs: https://www.npmjs.com/package/react-sortablejs#tag

![Screenshot (56)](https://github.com/EddieHubCommunity/BioDrop/assets/73095526/69ac79b7-d502-46b2-a074-7770666b1d44)

Reply:
Oh! thank you for the review! It should be okay by now! 
Let me know if any further adjustment is necessary